src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -262,33 +259,28 @@ def is_package_version_installed(self, package_name, package_version):
                 if 'Package: ' in line:
                     if package_name in line:
                         composite_found_flag = composite_found_flag | 1
-                    else:  # should never hit for the way this is invoked, hence telemetry
```

Comment:
Same comment all across -- telemetry exists independently. Composite logger chooses to utilize it.